Kickstarter-iOS/Views/Controllers/ChangePasswordViewController.swift

```diff
@@ -7,17 +8,51 @@ final class ChangePasswordViewController: UIViewController {
   @IBOutlet fileprivate weak var confirmNewPassword: UITextField!
```

Comment:
Just like you created the validationErrorMessageLabel and onePasswordButton, I would add `Field` (or `TextField` ) to the UITextField properties, making it easier to read.